### PR TITLE
Removed conn str and added exclude

### PR DESCRIPTION
**Pull request by jeremydvoss**
Update samples to rely on conn str env var. Updated samples that used to have instrumentation configs with env var instructions.

### CODE REVIEW

**Review by JacksonWeber — APPROVED** [review]

File: azure-monitor-opentelemetry/samples/tracing/http_fastapi.py [hunk 1]
```diff
@@ -26,6 +24,7 @@ async def exception():
     raise Exception("Hit an exception")
 
 
+# Set the OTEL_PYTHON_EXCLUDE_URLS environment variable to "http://127.0.0.1:8000/exclude"
```
Comment (lzchen) [inline azure-monitor-opentelemetry/samples/tracing/http_fastapi.py:27]: Should add this env var to the README

File: azure-monitor-opentelemetry/samples/logging/correlated_logs.py [hunk 2]
```diff
@@ -10,7 +10,7 @@
 from opentelemetry import trace
 
 configure_azure_monitor(
-    connection_string="<your-connection-string>",
+    # connection_string="<your-connection-string>",
```
Comment (lzchen) [inline azure-monitor-opentelemetry/samples/logging/correlated_logs.py:13]: What's the reason for this change?

File: azure-monitor-opentelemetry/samples/tracing/http_fastapi.py (hunk 1 above)
Comment (jeremydvoss) [reply azure-monitor-opentelemetry/samples/tracing/http_fastapi.py:27]: I guess we could. It's part of the instrumentation's readmes, but we did include it in the docs. There are other instrumentation related env vars I have not delt with much, so I think it would make sense to leave that for another PR where I add a whole section for instrumentation configuration env vars

**Review by lzchen — APPROVED** [review]
